### DISCUSSION TITLE
Feature request: Better BiDi support

### DISCUSSION THREAD

**Discussion by shaib** [pretalx/pretalx#1799] [Ideas] [answered]
No time now to make a proper bug report, but: In a multilingual system, where there is a RTL language, the multilingual forms should have the language-specific fields each have its own direction, instead of relying on the direction of the main interface language.

**Comment by rixx** [comment]
That's a good point – possibly best raised in [django-i18nfield](https://github.com/raphaelm/django-i18nfield/), whose `I18nFormField` and `I18nTextarea` we are using.

**Reply by rixx** [reply, marked as answer]
I also added the CSS overriding direction for Hebrew and Arab textareas/input fields in particular that pretix linked (ad536b2) – I think combined with the upcoming changes to django-i18nfield (thank you for raising this there!) we should be doing quite a bit better now?
